tests/Makefile.am

```diff
@@ -21,3 +23,7 @@ dist_noinst_SCRIPTS = \
 	$(test_scripts) \
 	t_cltsrv-down.sh
 
+.PHONY:	unit_tests
+unit_tests:
```

Comment:
This seems to not be needed.  `SUBDIRS = unit_tests` suffices.  (Also, there is no 'unit_tests' target in unit_tests/Makefile.am.)